AvatarChatbot/tests/test_compose_on_gaudi.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+echo "IMAGE_REPO=${IMAGE_REPO}"
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+if ls $LOG_PATH/*.log 1> /dev/null 2>&1; then
+    rm $LOG_PATH/*.log
+    echo "Log files removed."
+else
+    echo "No log files to remove."
+fi
+ip_address=$(hostname -I | awk '{print $1}')
+
+
+function build_docker_images() {
+    cd $WORKPATH
+    git clone https://github.com/opea-project/GenAIComps.git
+    cd GenAIComps
+
+    docker build -t opea/whisper-gaudi:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/asr/whisper/dependency/Dockerfile.intel_hpu .
```

Comment:
We should use `docker compose -f build.yaml build ${service_list} --no-cache > ${LOG_PATH}/docker_image_build.log` to build test images, refer to https://github.com/opea-project/GenAIExamples/blob/c7e33647ada2b403ac409e97f79c726531f0d51d/ChatQnA/tests/test_compose_on_gaudi.sh#L22

Reply:
Hi @chensuyue. Changes made in the commit https://github.com/ctao456/GenAIExamples/commit/df6f9765a9853c2db26c626c8fce4db8a3b97be7
Please review thanks.